src/pages/homepage/Global.Style.js

```diff
@@ -0,0 +1,93 @@
+import { createGlobalStyle } from "styled-components";
```

Comment:
use this component in  main component (app.js)

Reply:
eventually removed global styles